data/DaduDev.json

```
@@ -2,15 +2,24 @@
   "name": "Shaik Dadapeer",
   "type": "public",
   "bio": "Computer Science student just starting out with open source and soon be landing on a remote job",
+  "tags": [
+    "Open Source",
```

```suggestion
    "OpenSource",
```

📋 Usually, spaces break tags. It's easier to adopt a lizard-brain approach and just "skip the spaces". EddieHub is space-software so it might actually support it, but when you see `tags` skip the spaces.